libs/cart/src/facades/cart/cart-facade.interface.ts

```diff
@@ -37,7 +37,13 @@ export interface DaffCartFacadeInterface<
   shippingInformation$: Observable<DaffCart['shipping_information']>;
   availableShippingMethods$: Observable<DaffCart['available_shipping_methods']>;
   availablePaymentMethods$: Observable<DaffCart['available_payment_methods']>;
+
   isCartEmpty$: Observable<boolean>;
+  /**
+   * Whether the cart's shipping address equals the billing address.
+   * Returns false when they are both nullity.
```

Comment:
isn't nullity more of a law term? Could we stick with just "null"?

Reply:
Well like falsey can be false, undefined, or null, nullity can be null or undefined. I think thats the term for it

Reply:
How about `Returns false if either address is null or undefined`?

Reply:
oh ok. I'd never heard it before

Reply:
I think I like that better. The only thing I can find about "nullity" is a math term for what you get when you divide by zero. If you can find anything CS related, send it my way

Reply:
I guess it might be more slang than an actual term. A google search for `"nullity" js null or undefined` does return a few results but its probably best to be explicit in the docs.

Reply:
changed